### PR TITLE
Removed draft PR status

### PR DESCRIPTION
**Pull request by shanejonas**
proposing this change since [personal private repos are not allowed to use `draft` pull requests](https://github.community/t/draft-pull-requests-are-not-available-on-our-private-repositories-in-last-few-days/2466)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -32,4 +36,5 @@ runs:
           ${{ steps.update-packages.outputs.NEW_VERSION }} \
           ${{ inputs.release-branch-prefix }} \
           ${{ github.actor }} \
+          ${{ inputs.create-draft-pr }} \
           ${{ inputs.artifacts-path }}
```
Comment (shanejonas) [inline action.yml:40]: this changes the order because artifacts-path is optional

File: scripts/create-release-pr.sh [hunk 2]
```diff
@@ -20,7 +20,8 @@ fi
 
 ACTION_INITIATOR="${3}"
 
-ARTIFACTS_DIR_PATH="${4}"
+CREATE_DRAFT_PR="${4}"
+ARTIFACTS_DIR_PATH="${5}"
```
Comment (shanejonas) [inline scripts/create-release-pr.sh:24]: see above on why the order of args changed

File: action.yml [hunk 3]
```diff
@@ -15,6 +15,10 @@ inputs:
   artifacts-path:
     description: 'The path to the directory where this action will create its artifact files.'
     required: false
+  create-draft-pr:
```
Comment (rekmarks) [inline action.yml:18]: I think the most clear option would be to rename this as follows:
```suggestion
  created-pr-status:
```

We can then keep the Bash variables you created the same and parse this input to true/false, or we can rename the variables as well.

**Review by rekmarks — CHANGES_REQUESTED** [review]

**Review by rekmarks — APPROVED** [review]
LGTM!

**Pull request merged**